typescript/src/deposit.ts

```diff
@@ -14,11 +14,11 @@ import { Bridge, Identifier } from "./chain"
 const { opcodes } = bcoin.script.common
 
 /**
- * Duration of the deposit refund locktime in seconds. After that time, the
- * depositor can make a refund of an unswept deposit using the refund public
+ * Default duration of the deposit refund locktime in seconds. After that time,
+ * the depositor can make a refund of an unswept deposit using the refund public
  * key.
  */
-export const DepositRefundLocktimeDuration = 2592000 // 30 days
+export const DefaultDepositRefundLocktimeDuration = 2592000 // 30 days
```

Comment:
Shouldn't we set the default to 90 days instead?

Reply:
A lot of tests rely on 30 days. Using an optional value is also more flexible. Last but not least, 90 days is rather temporary.

Reply:
How big is the impact on tests? 

I think we can use an optional value but we should set the default to what is currently expected by the system. The smart contract will not allow revealing anything that is 8 months and 2 weeks ahead of the unlock. 

8 months and 2 weeks is a value for the next few months, it's is not that temporary.

If we leave this as 30 days and optional, external devs using `tbtc-v2.ts` will produce non-revealable deposits from their applications. The same will happen in case we forget to use a non-default value in the dApp.

Reply:
>  In that case, I think there should not be any default in `tbtc-v2.ts` and we should always expect an explicit value.

Makes sense!